lib/exandra/connection.ex

```diff
@@ -785,10 +791,44 @@ defmodule Exandra.Connection do
     |> Enum.map(fn {_, name, _, opts} -> %{name: name, opts: opts} end)
   end
 
-  def column_definitions([]), do: raise(RuntimeError, "you must define at least one column")
+  def alter_column_definitions([]), do: raise(RuntimeError, "you must define at least one column")
```

Comment:
Nit: `RuntimeError` is what you raise when you just call `raise/1` with a string

```suggestion
  def alter_column_definitions([]), do: raise("you must define at least one column")
```